### PR TITLE
README: Update CLI docs

### PR DESCRIPTION
**Pull request by ayan-b**

### CODE REVIEW

**Comment by yunhailuo** [comment]
This is great! I think this is ready. But let me look into entry points and `gdc_check_new` a little bit.

**Comment by ayan-b** [comment]
@yunhailuo Update the docs according to new CLI, please review.

**Comment by yunhailuo** [comment]
Sorry, I was confused and not realize "Basic usage with command line tools" is actually for "Command line utilities". So I guess there is not need for a new section, right?

And `gdc2xena` is not an independent entry point anymore. We need to fix that (First part in "Basic usage with command line tools").

**Comment by ayan-b** [comment]
@yunhailuo, please take a look now. :)

File: README.rst [hunk 1]
```diff
@@ -254,11 +262,11 @@ Advanced usage with XenaDataset and its subclasses
   
   \1. GDC API Available File Fields: https://docs.gdc.cancer.gov/API/Users_Guide/Appendix_A_Available_Fields/#file-fields
   
-  - **Customize** ``TCGAPhenoset``
+  - **Customize** ``GDCPhenoset``
```
Comment (yunhailuo) [inline README.rst]: This title should be just `**for TCGA projects**`. And we can have a title below for TARGET.

File: README.rst [hunk 2]
```diff
@@ -273,12 +281,15 @@ Advanced usage with XenaDataset and its subclasses
       
     - Customize ``metadata`` step
     
-      Different from ``download`` and ``transform``, there is no special settings for the ``metadata`` method of ``TCGAPhenoset``. Therefore, similar to that of ``GDCOmicset``, this step can be customized through ``metadata_template``, ``metadata_vars`` and ``gdc_release`` properties. And to call just the ``metadata`` method, an existing ``matrix`` is enough.
-    
-  - **Customize** ``TARGETPhenoset``
```
Comment (yunhailuo) [inline README.rst]: We can still have a title here: `**Customize** ``GDCPhenoset`` **for TARGET projects**`

File: README.rst [hunk 3]
```diff
@@ -195,23 +203,23 @@ Advanced usage with XenaDataset and its subclasses
 
 .. _subclasses:
 
-- **Build GDC importing pipelines with** ``GDCOmicset``, ``TCGAPhenoset``, ``TARGETPhenoset`` **or** ``GDCSurvivalset`` **classes**
+- **Build GDC importing pipelines with** ``GDCOmicset``, ``GDCPhenoset`` **or** ``GDCSurvivalset`` **classes**
```
Comment (yunhailuo) [inline README.rst]: Can you also check there is no TCGAPhenoset or TARGETPhenoset in the code base anymore? I saw some here: https://github.com/yunhailuo/xena-GDC-ETL/blob/master/xena_gdc_etl/xena_dataset.py#L5-L6 Should we change them as well?

**Comment by yunhailuo** [comment]
Seems valid for rebase and merge. Make sure the commit history is good before merge ready.

**Comment by yunhailuo** [comment]
Changed "Merge ready label" to "ready for merge review". Hopefully it's useful.

File: README.rst (hunk 3 above)
Comment (ayan-b) [reply README.rst]: Yup, they should be changed as well.

File: README.rst [hunk 4]
```diff
@@ -273,12 +282,17 @@ Advanced usage with XenaDataset and its subclasses
...
+
+  - **Customize** ``GDCPhenoset`` for **TARGET projects**
+
+    TARGET phenotype data for Xena contains only the clinical data (no
+    biospecimen data), as `detailed below <#transform-phenotype>`_. The importing
+    process is quite similar to that of a ``GDCOmicset``. You can customize
+    ``GDCPhenoset`` with ``download_map``, ``read_raw``, ``raws2matrix``,
+    ``metadata_template``, ``metadata_vars`` and ``gdc_release`` in the same way
+    as that of `GDCOmicset <#customize-gdcomicset>`_.
+
```
Comment (yunhailuo) [inline README.rst]: Why so many new lines here? They are the same paragraph like above (line #285)?

Comment (ayan-b) [reply README.rst]: The line was quite long (horizontally), so I broke them and forgot to undo the change. Changing right away!

**Comment by yunhailuo** [comment]
Do you want to squash the last two commits so we do rebase and merge here? Or do you want to do just squash and merge for this PR? @ayan-b

**Comment by ayan-b** [comment]
I think `Squash` all commits would be good, since 344f6444e48e95af9addd318e5046d0dd64251bf isn't really valid anymore.

**Pull request merged**